Source/Core/Common/StringUtil.cpp

```diff
@@ -77,13 +77,10 @@ bool AsciiToHex(const std::string& _szValue, u32& result)
   // Set errno to a good state.
   errno = 0;
 
-  char* endptr = nullptr;
-  const u32 value = strtoul(_szValue.c_str(), &endptr, 16);
+  const u32 value = strtoul(_szValue.c_str(), nullptr, 16);
 
-  if (!endptr || *endptr)
-    return false;
-
-  if (errno == ERANGE)
+  // Fail if errno EINVAL or ERANGE
```

Comment:
I checked those sources also and because there are two different version. So I went to check the standard. Indeed it is not saying anything about EINVAL so I need to change my source :D.

> Source Open STD page 344 -->